### PR TITLE
6304 - allow product developers to add custom class in event labels

### LINKED ISSUE #6304: allow product devs to add custom css class to event labels in calendar component
**Is your feature request related to a problem or use case? Please describe.**
A clear and concise description of what the problem is. [e.g. {Component Name} has a problem when {...}]

New release has removed custom classes from getting passed into event label in calendar component, thus broke some of the pages that requires customization.

Please refer to code here:
https://github.com/infor-design/enterprise/commit/74d508d69cb34b3fe524e415766c947c5a4ad2b6

**Describe the solution you'd like**
A clear and concise description of what you want to happen. [e.g. {Component Name} would work better if {...}]
As per discussion with Tim, I would like to suggest that we add an additional event attribute "cssClass" to allow product devs to add custom class names. 
https://github.com/infor-design/enterprise/blob/4.62.0/src/components/calendar/calendar.js#L824

**Describe alternatives you've considered**

Might need to write jQuery hacks to add css classes to each event label if we cannot allow this solution.

**Additional context**
And what application do you work on? [e.g. {Infor XYZ}]

### PR DESCRIPTION
**Pull request by edinfor**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

**Related github/jira issue (required)**:

Closes #6304 

**Steps necessary to review your pull request (required)**:

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 ## v4.63.0
 
 ## v4.63.0 Fixes
-
+- `[Calendar]` Allow product devs to add custom css class to event labels in calendar component. ([#6304](https://github.com/infor-design/enterprise/issues/6304))
+- `[Listview]` Change the font size of heading, subheading, and micro in Listview Component. ([#4996](https://github.com/infor-design/enterprise/issues/4996))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:7]: Maybe some merge conflict here? Also please keep alphabatized

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Looks like a small problem in the log and two tests failing

1. https://github.com/infor-design/enterprise/runs/6029328273?check_suite_focus=true#step:8:2870 -> could be anything you did?
2. Can you try and update with main. I think i fixed this one https://github.com/infor-design/enterprise/runs/6029328273?check_suite_focus=true#step:8:2874 . If it doesnt work after then you can skip this test for now and i'll fix it later? (by changing it to`xit`)

**Comment by tmcconechy** [comment]
Something seems to have happened with the tests https://github.com/infor-design/enterprise/runs/6062552276?check_suite_focus=true#step:8:2938 @edinfor

**Comment by tmcconechy** [comment]
ok the masthead one is "flaky" can you set it to skip (xit) for now and i'll look later? Thanks

**Comment by edinfor** [comment]
https://github.com/infor-design/enterprise/runs/6064274729?check_suite_focus=true#step:11:74

any idea why this failed?

**Comment by tmcconechy** [comment]
Also flaky intermittent i guess

**Comment by ericangeles** [comment]
Looks like a flaky test, I re-run the build btw.

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**